docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -1367,6 +1370,60 @@ and annotating it with `@PersistenceUnitExtension` (or `@PersistenceUnitExtensio
 will replace the current Quarkus default implementation `io.quarkus.hibernate.orm.runtime.tenant.DataSourceTenantConnectionResolver`.
 Your custom connection resolver would allow for example to read tenant information from a database and create a connection per tenant at runtime based on it.
 
+[CAUTION]
+====
+<<automatic-integration,Automatic integrations>> will **not** work with programmatically-created `ConnectionProvider`.
+This implies all these integrations, including those from other Quarkus modules (e.g. xref:smallrye-health.adoc[SmallRye Health]) must be done manually.
+====
+
+Here is an implementation example of a `TenantConnectionResolver` implementation using standard Quarkus technologies – Agroal for pooling and Narayana for transactions:
+
+[source,java]
+----
+@ApplicationScoped
+@PersistenceUnitExtension
+public class ExampleTenantConnexionResolver implements TenantConnectionResolver {
```

Comment:
`Connexion` => `Connection` -- the spelling mistake appears multiple times below, not just here.

Reply:
Done